packages/maskbook/src/plugins/Wallet/UI/ImportWalletDialog/ImportResult.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, makeStyles, Table, TableBody, TableCell, TableHead, TableRow, Typography } from '@material-ui/core'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+import { useStylesExtends } from '../../../../components/custom-ui-helper'
+
+const useStyles = makeStyles((theme) => ({
+    content: {
+        padding: theme.spacing(4, 5, 1),
+    },
+    title: {
+        fontSize: 18,
+        fontWeight: 500,
+        color: '#15181B',
+        padding: theme.spacing(0, 1),
+    },
+    dialogActions: {
+        alignItems: 'center',
+        padding: theme.spacing(3, 5),
+    },
+    headCell: {
+        borderBottom: 'none',
+        backgroundColor: '#F3F3F4',
+    },
+    bodyCell: {
+        borderBottom: 'none',
+        padding: '0 10px',
+    },
+}))
+
+const addressList = [
```

Comment:
What are those mock data for?

Reply:
Removed in https://github.com/DimensionDev/Maskbook/pull/3298/commits/ceb17bf35a934b93a06b2c82020d6bb7e973e963